ToNote/Controls/TodoControl.cs

```diff
@@ -36,6 +36,38 @@ public TodoControl(Todo todo)
                     this.BackspacePressedWhileEmpty?.Invoke(this, new RoutedEventArgs());
                 };
             };
+
+            this.AllowDrop = true;
+
+            this.PreviewMouseMove += (s, e) =>
+            {
+                var point = e.GetPosition(this);
+
+                if (point.X <= 5)
```

Comment:
Sane here, ternary would be better, `Mouse.OverrideCursor = point.X <= 5 ? Cursors.SizeAll : null;`